.github/workflows/docker.yml

```diff
@@ -0,0 +1,43 @@
+name: Docker
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
Do we need to run this action on every push? Could it be part of the `release.yml` workflow instead?